### PR TITLE
Support Oreo notifications

### PR DESCRIPTION
**Pull request by cmaitchison**
Android Oreo (API 26) requires notification channels to be created and used with notifications.

This PR creates a default notification channel and uses it when sending local notifications.

### CODE REVIEW

File: android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import android.util.Log;
+
+import static com.wix.reactnativenotifications.Defs.LOGTAG;
+
+public class OreoNotifications {
+
+    public static void initialize(Context context) {
+        Log.d(LOGTAG, "OreoNotifications.initialize()");
+
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.O) return;
```
Comment (shockdesign) [inline android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java:18]: As noted this comparison should be flipped < instead of >

File: android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        return getChannelId(context);
+    }
+
+    public static String getChannelId(Context context) {
+        return context.getPackageName() + "_default";
+    }
+
+    public static void setChannel(Context context, Notification.Builder builder) {
+        Log.d(LOGTAG, "OreoNotifications.setChannel()");
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.O) return;
```
Comment (shockdesign) [inline android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java:33]: As noted this comparison should be flipped < instead of >

Comment (cmaitchison) [reply android/src/main/java/com/wix/reactnativenotifications/core/notifications/OreoNotifications.java:33]: That's weird. I pushed the fix to this branch and merged to master this arvo. The PR diff must become immutable after merge.

TL;DR Fixed and merged.
